packages/webapp/src/pages/ConnectionList.tsx

```diff
@@ -142,8 +142,8 @@ export default function ConnectionList() {
         const value = e.target.value;
         setSelectedIntegration(value);
         setLoaded(false);
-
-        navigate(location.pathname + '?' + queryString.stringify({ ...queryParams, integration: value }));
+        setOffset(0);
```

Comment:
This aims to prevent a scenario where the filtered result differs in `offset` value from the one requested to the server. This is also a bug in activity list of which I am working on a pull request for it as I also show integration and connection filter options which are only present in the activity list, as you had suggested earlier on connection list![activity](https://github.com/NangoHQ/nango/assets/85742599/5f0f7f1c-0fc9-45ca-b895-358cc7c940b1)